src/features/courses/course-controller.ts

```diff
@@ -3683,8 +3683,9 @@ You should be able to reply to the student's email address (${options.student.pr
                 data.questions[i].grades[j].lastInfluencingAttempt.studentGradeInstance.problemAttachments = probAttach.map(pA => pA.get({plain: true}));
             })
         );
-
-        return {user: user, topic: data};
+        
+        const baseUrl = configurations.attachments.baseUrl;
+        return {user: user, topic: data, baseUrl};
```

Comment:
You didn't have to change any types for the result?

Reply:
Yeah, bad things were done. 
![image](https://user-images.githubusercontent.com/68256772/98600793-1e1f0200-22ac-11eb-9382-c61598868c6a.png)